tests/phpunit/PropertySuggester/Suggesters/SimpleSuggesterTest.php

```diff
@@ -68,7 +68,7 @@ public function testSuggestByPropertyIds() {
 	}
 
 	public function testSuggestByItem() {
-		$item = Item::newEmpty();
+		$item = new Item();
 		$item->setId( new ItemId( 'Q42' ) );
```

Comment:
done, though not going to touch more of this code.  should be a separate task to update code for new data model stuff.